stint/io.nim

```diff
@@ -229,7 +231,9 @@ func readDecChar(c: range['0'..'9']): int {.inline.}=
   # specialization without branching for base <= 10.
   ord(c) - ord('0')
 
-func parse*[bits: static[int]](input: string, T: typedesc[StUint[bits]], radix: static[uint8] = 10): T =
+func parse*[bits: static[int]](input: string,
+                               T: typedesc[StUint[bits]],
+                               radix: static[uint8] = 10): T =
```

Comment:
This is the kind of function that shouldn't be made inline in my opinion.

There is a ``while`` loop inside.

Reply:
Yup, I didn't add push inline to that function.